data/users.js

```diff
@@ -1,8 +1,65 @@
+const { ObjectID } = require('bson');
 const mongoCollections = require('../config/mongoCollections');
+const users = mongoCollections.users;
+const bcrypt = require('bcrypt');
 const getUsersCollection = mongoCollections.users;
-const validate = require('../validations/data');
 const validations = require("../validations/validations");
 const {ObjectId} = require("mongodb");
+const validate = require('../validations/data');
+
+async function getPersonById(id){
+    // In this function we need to find the user for a given id and return them.
```

Comment:
Validation of the input "id" is missing here, I guess! 
You can use either validations.validateId(id) or validate.convertID(id).